ROADMAP.md

```diff
@@ -8,31 +8,64 @@ Each weekly release will include necessary tasks that lead to the completion of
+From [November through January](docs/roadmaps/2020_November_to_January.md), we added support for (among other things):
 
-From October-January ‘21, we will be prioritizing the following areas of work:
+- SSO Permission Sets
+- EC2 Managed Prefix Lists
+- Firewall Manager Policies
+- SASL/SCRAM Authentication for MSK
+- ImageBuilder
+- LakeFormation
+- Serverless Application Repository
+- Cloudwatch Composite Alarms
+
+As well as partnering with AWS to provide launch day support for:
+
+- Network Firewall
+- Code Signing for Lambda
+- Container Images for Lambda
+- Gateway Load Balancer
+- Spot Launch for EKS Managed Node Groups
+
+From February-April ‘21, we will be prioritizing the following areas of work:
+
+## Provider Functionality: Default Tags
+
+Issue: [#7926](https://github.com/hashicorp/terraform-provider-aws/issues/7926)
+
+Default Tags builds on the workflows in Ignore Tags to provide additional control over the ways Terraform manages tagging capabilities. Users will be able to specify lists of tags to apply to all resources in a configuration at the provider level. Our goal in offering this use case is to assist in tidying up configuration files, decreasing development efforts, and streamlining cost allocation and resource attribution within organizations of all sizes. 
 
 ## New Services
 
-### AWS SSO Permission Sets
-Issue: [#15108](https://github.com/hashicorp/terraform-provider-aws/issues/15108)
+### CloudWatch Synthetics
+Issue: [#11145](https://github.com/hashicorp/terraform-provider-aws/issues/11145)
 
-_[AWS SSO](https://docs.aws.amazon.com/singlesignon/latest/APIReference/welcome.html) account assignment APIs enable you to build automation to create and update permissions that align with your company's common job functions. You can then assign the permissions to users and groups to entitle them for access in their required accounts. For example, you can give your developers broad control over resources in developer accounts, and limit that control to authorized operations personnel in production accounts. The new AWS CloudFormation support enables you to automate account assignments as you build new accounts. You can also use the APIs to decode user and group names from the unique identifiers that appear in AWS CloudTrail logs._
+_[CloudWatch Synthetics](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Canaries.html) You can use Amazon CloudWatch Synthetics to create canaries, configurable scripts that run on a schedule, to monitor your endpoints and APIs. Canaries follow the same routes and perform the same actions as a customer, which makes it possible for you to continually verify your customer experience even when you don't have any customer traffic on your applications. By using canaries, you can discover issues before your customers do._
 
 Support for AWS SSO Permission Sets will include:
```

Comment:
I think this should be `CloudWatch Synthetics`?

Reply:
Thanks! Good Catch - i'll fix it!